controllers/snapshot/snapshot_adapter_test.go

```diff
@@ -32,6 +32,7 @@ var _ = Describe("Snapshot Adapter", Ordered, func() {
 		hasApp                  *applicationapiv1alpha1.Application
 		hasComp                 *applicationapiv1alpha1.Component
 		hasSnapshot             *applicationapiv1alpha1.ApplicationSnapshot
+		hasSnapshot1            *applicationapiv1alpha1.ApplicationSnapshot
```

Comment:
maybe renaming this variable to something relevant to the `pull_request` ? 
Something like `hasSnapshotPullRequest` ?

Reply:
Sure. Updated to  "hasSnapshotPR".